dapps/marketplace/src/pages/listings/Listings.js

```diff
@@ -40,7 +40,14 @@ class Listings extends Component {
     this.state = {
       first: 12,
       search: getStateFromQuery(props),
-      sort: 'featured'
+      // currently sort is activated through hardcoding the sort state
+      // order = 'asc' or 'desc'
+      // I have made it possible to sort by other values but its not tested
+      // target: 'price.amount' is the current goal
+      // this will be updated to be user set when the front end is completed
+      // graphql is expecting string values so to disable use empty strings
+      sort: { target: '', order: '' }
```

Comment:
Could we just have a var for 'sort' and another for 'order' instead of nesting those properties inside a 'sort' object?

Reply:
yea that is better, it makes the front end simpler as well. I'll change it.